avocado/core/job.py

```diff
@@ -471,12 +471,16 @@ def _run(self):
         self._setup_job_results()
         self.__start_job_logging()
 
-        try:
-            self.test_suite = self._make_test_suite(self.urls)
-        except loader.LoaderError as details:
-            stacktrace.log_exc_info(sys.exc_info(), 'avocado.app.debug')
-            self._remove_job_results()
-            raise exceptions.OptionValidationError(details)
+        if (getattr(self.args, 'remote_hostname', False) and
+           getattr(self.args, 'remote_no_copy', False)):
```

Comment:
this only covers the `remote-hostname` and not the rest of the remote plugins. (`avocado run /tmp/passtest.py --vm-domain fedora22-2 --vm-no-copy` fails while `avocado run /tmp/passtest.py --remote-hostname test --remote-no-copy` works well)